chain/default_store_test.go

```diff
@@ -34,14 +34,14 @@ func initStoreTest(ctx context.Context, t *testing.T, dstP *DefaultSyncerTestPar
 	requireSetTestChain(t, con, true, dstP)
 }
 
-func newChainStore(dstP *DefaultSyncerTestParams) chain.Store {
+func newChainStore(dstP *DefaultSyncerTestParams) *chain.DefaultStore {
```

Comment:
This should be renamed to `chain.Store`. It was named `DefaultStore` as an alternative to `StoreImpl`, which is the (bad) Java convention.